### ISSUE TITLE
save training and resume later

### ISSUE THREAD

**Issue by Hunterwolf88** [adeshpande3/Facebook-Messenger-Bot#26]
Is it possible to resume a previously started training on the same model?
How to stop training and save progress?

**Comment by adeshpande3** [comment]
I believe the current code can already let you do that. In lines 245-246 in Seq2Seq.py, you'll see that we're saving the model every 10,000 iterations. Then, in line 201, you see that we call the restore function. So, when you restore the model, the model with the previously updated/trained weights will be loaded in and then you can start your training loop again.

**Comment by Hunterwolf88** [comment]
effectively I have some pretrained_seq2seq.ckpt files, from "10000" until "70000" (I assume from the past training session I interrupted).
But if I simply run "Seq2Seq.py" it starts from current loss: 12.5, iteration 0.
in model's folder "checpoint file" I find:
`model_checkpoint_path: "pretrained_seq2seq.ckpt-10000"
all_model_checkpoint_paths: "pretrained_seq2seq.ckpt-10000"`

**Comment by Hunterwolf88** [comment]
uncommented line 201 in Seq2Seq.py
commented line 202.

`saver.restore(sess, tf.train.latest_checkpoint('models/'))
#sess.run(tf.global_variables_initializer())`

thanks :)
